app/style/common/label.less

```diff
@@ -48,6 +48,9 @@
   font-weight: @font-weight-bold;
 }
 
-.label-username:not(:empty):before {
-  content: "@";
+.label-username {
+  .text-selection;
+  &:not(:empty):before {
+    content: "@";
```

Comment:
I noticed that we have a mix of quotes.. in our styles :) Maybe switch all to single quotes?